src/client/containers/CartPage/CartPage.Container.js

```diff
@@ -66,54 +93,89 @@ const CartPageContainer = ({ isAuthenticated }) => {
+          userId={userId}
+          isFavorite={checkFavortie(item.productId)}
+        />
+      );
+    });
+  };
+  const showDeliveryInfo = () => {
+    if (userId) {
+      return <DeliveryInfoV2 editMode={true} vertDisplay={false} user={user} />;
+    }
+  };
+  const showContactDetails = () => {
+    if (userId) {
+      return (
+        <ContactForm
+          fullName={auth.currentUser.displayName}
+          email={auth.currentUser.email}
+        />
+      );
+    }
+  };
+
   return (
-    <div>
-      <h1>SHOPPING CART</h1>
-      <div className="main">
-        <div className="left">
-          <div className="item-list">
-            {IsLoading ? (
-              <Loader />
-            ) : (
-              cartItem.map((item, index) => {
-                return (
-                  <ShoppingItem
-                    productName={item.name}
-                    quantity={item.stock_amount}
-                    price={item.price}
-                    productImg={item.picture}
-                    initValue={item.quantity}
-                    getQuantity={(value) => {
-                      getItemQuantity(index, value);
-                    }}
-                    onDelete={() => handleOnDeleteItem(index)}
-                  />
-                );
-              })
-            )}
-          </div>
-        </div>
-        <div className="right">
-          <div className="total-price">
-            <TotalPrice subTotal={total} />
-          </div>
-          <div className="contact">
-            <ContactForm
-              fullName={
-                isAuthenticated ? `${auth.currentUser.displayName}` : 'Guest'
-              }
-              email={isAuthenticated ? `${auth.currentUser.email}` : ''}
-            />
+    <div className="cart-page">
+      <div className="header">
+        <h1>SHOPPING CART</h1>
+      </div>
+      <div className="cart-page-containter">
+        <div className="top">
+          <div className="left">
+            <div className="cart-items">
+              {isLoading ? <Loader /> : showCartItems()}
+            </div>
+            <div className="delivery-info">
+              {userId && userInfo.isLoading ? <Loader /> : showDeliveryInfo()}
+            </div>
           </div>
-          <div>
-            <div className="review-btn">
-              <ButtonComponent title="REVIEW ORDER" />
+          <div className="right">
+            <div className="total">
+              <TotalPrice subTotal={total} />
+            </div>
+            <div className="contact">
+              {userId && userInfo.isLoading ? <Loader /> : showContactDetails()}
             </div>
-            <div className="shopping-btn">
-              <ButtonComponent
-                title="KEEP SHOPPING"
-                onClick={() => history.push('/')}
-              />
+            <div className="buttons">
+              <div className="review-btn">
+                <ButtonComponent title="REVIEW ORDER" />
```

Comment:
As we talked before, I think this page needs to  **post** the **order** to **db**  by clicking this **Review Order** button.

Reply:
yes, agree... but this is part of #239 . So I wont include that in this PR. @semih1239 and @BikramRawat  are working on that